### ISSUE TITLE
Lossless convertion?

### ISSUE THREAD

**Issue by ecimen** [AydinAdn/MediaToolkit#97]
İts possible to enable lossless  codec with MediaToolkit?

**Comment by kcvinker** [comment]
Yeah, If its possible, it will be great. Bcause, tools like Moo Video cutter are cutting videos very fast with lossless cutting. It took 140 seconds in my pc to cut a 600 second clip (14957 frames).
